Formula/arm-linux-gnueabihf-binutils.rb

```diff
@@ -1,10 +1,10 @@
 class ArmLinuxGnueabihfBinutils < Formula
   desc "FSF/GNU binutils for cross-compiling to arm-linux"
   homepage "https://www.gnu.org/software/binutils/binutils.html"
-  url "https://ftp.gnu.org/gnu/binutils/binutils-2.35.1.tar.xz"
-  mirror "https://ftpmirror.gnu.org/binutils/binutils-2.35.1.tar.xz"
-  sha256 "3ced91db9bf01182b7e420eab68039f2083aed0a214c0424e257eae3ddee8607"
-  license "GPL-2.0"
+  url "https://ftp.gnu.org/gnu/binutils/binutils-2.36.tar.xz"
+  mirror "https://ftpmirror.gnu.org/binutils/binutils-2.36.tar.xz"
+  sha256 "5788292cc5bbcca0848545af05986f6b17058b105be59e99ba7d0f9eb5336fb8"
+  license "GPL-3.0-or-later"
```

Comment:
Since it builds from the same source, maybe it needs the same license as `binutils`?

https://github.com/Homebrew/homebrew-core/blob/9298b219c40214ca661f0ecc22cb85979a108e3b/Formula/binutils.rb#L7

Reply:
>    This file is part of GNU Binutils.

>    This program is free software; you can redistribute it and/or modify
   it under the terms of the GNU General Public License as published by
   the Free Software Foundation; either version 3 of the License, or
   (at your option) any later version.
   
   
I think the license should be `GPL-3.0-or-later`